src/core/function_app.tf

```diff
@@ -160,6 +160,12 @@ module "app_snet" {
   }
 }
 
+data "azurerm_subnet" "ioweb_profile_snet" {
+  name                 = format("%s-ioweb-profile-snet", local.project)
```

Comment:
location `weu` is missing here

Reply:
fixed in https://github.com/pagopa/io-infra/pull/679/commits/68d692316214b8cd97fc28da3195bdf7f818de2c